[2023-11-21T00:38:53.252+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [queued]>
[2023-11-21T00:38:53.257+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [queued]>
[2023-11-21T00:38:53.257+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:38:53.264+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-17 00:00:00+00:00
[2023-11-21T00:38:53.273+0000] {standard_task_runner.py:57} INFO - Started process 92 to run task
[2023-11-21T00:38:53.277+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-17T00:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpylo51djy']
[2023-11-21T00:38:53.280+0000] {standard_task_runner.py:85} INFO - Job 117: Subtask load_data
[2023-11-21T00:38:53.323+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [running]> on host c6d84631e1b9
[2023-11-21T00:38:53.378+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-17T00:00:00+00:00'
[2023-11-21T00:38:56.715+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:38:56.798+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231117T000000, start_date=20231121T003853, end_date=20231121T003856
[2023-11-21T00:38:56.858+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:38:56.977+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21T00:44:28.203+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [queued]>
[2023-11-21T00:44:28.265+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [queued]>
[2023-11-21T00:44:28.272+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:44:28.358+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-17 00:00:00+00:00
[2023-11-21T00:44:28.391+0000] {standard_task_runner.py:57} INFO - Started process 186 to run task
[2023-11-21T00:44:28.419+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-17T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpswqpk7pj']
[2023-11-21T00:44:28.440+0000] {standard_task_runner.py:85} INFO - Job 169: Subtask load_data
[2023-11-21T00:44:28.648+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [running]> on host 51d09cf321f5
[2023-11-21T00:44:28.971+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-17T00:00:00+00:00'
[2023-11-21T00:44:32.644+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:44:32.717+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231117T000000, start_date=20231121T004428, end_date=20231121T004432
[2023-11-21T00:44:32.880+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:44:32.945+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21T01:00:26.529+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [queued]>
[2023-11-21T01:00:26.534+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [queued]>
[2023-11-21T01:00:26.535+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T01:00:26.542+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-17 00:00:00+00:00
[2023-11-21T01:00:26.551+0000] {standard_task_runner.py:57} INFO - Started process 193 to run task
[2023-11-21T01:00:26.555+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-17T00:00:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp21o2w_3e']
[2023-11-21T01:00:26.559+0000] {standard_task_runner.py:85} INFO - Job 229: Subtask load_data
[2023-11-21T01:00:26.647+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-17T00:00:00+00:00 [running]> on host e39dda9a9deb
[2023-11-21T01:00:26.741+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-17T00:00:00+00:00'
[2023-11-21T01:00:30.004+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T01:00:30.029+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231117T000000, start_date=20231121T010026, end_date=20231121T010030
[2023-11-21T01:00:30.099+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T01:00:30.133+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
